includes/stores/singlestore.php

```diff
@@ -115,14 +126,14 @@ function get_tokens( $args = array() ) {
 			$query['meta_value'] = $args['service'];
 		}
 
-		$token_type = 'request' == $args['type'] ? 'Keyring_Request_Token' : 'Keyring_Access_Token';
-		$tokens = array();
-		$posts = get_posts( $query );
+		$token_type = 'request' === $args['type'] ? 'Keyring_Request_Token' : 'Keyring_Access_Token';
```

Comment:
In the case where $args['type'] is set to 0 or TRUE then this would change the logic. I suppose it is possible that some instances of keyring are relying on this broken behaviour if they are calling get_tokens with these values, but that seems pretty unlikely. Because this is a .org plugin there's no way to know for sure :)

Reply:
That would be super broken. I don't think there's a reason to assume it might exist.